packages/angular/tsconfig.json

```diff
@@ -2,19 +2,35 @@
   "compileOnSave": false,
   "compilerOptions": {
     "baseUrl": "./",
-    "outDir": "./dist/out-tsc",
+	"outDir": "./dist",
+	"rootDir": "src",
     "sourceMap": true,
-    "declaration": false,
-    "moduleResolution": "node",
+	"inlineSources": true,
+    "declaration": true,
+	"moduleResolution": "node",
+	"module": "es2015",
     "emitDecoratorMetadata": true,
-    "experimentalDecorators": true,
-    "target": "es5",
+	"experimentalDecorators": true,
+	"downlevelIteration": true,
+	"importHelpers": true,
+    "target": "es2015",
```

Comment:
I think there are some mixed indentation types in this file

Reply:
🤦‍♂